three.js/examples/nft.html

```diff
@@ -128,7 +128,6 @@
 
         // create atToolkitContext
         var arToolkitContext = new THREEx.ArToolkitContext({
-            cameraParametersUrl: '../../data/data/camera_para.dat',
```

Comment:
deafult parameters are correct now, so it's not necessary to set up the default path anymore.